variables.tf

```diff
@@ -69,3 +69,15 @@ variable "vpc_id" {
   default     = null
   description = "An ID of the existing AWS VPC"
 }
+
+variable "create_zone" {
+  type        = bool
+  default     = true
+  description = "Set true or false to create new zone or use existing one"
+}
+
+variable "zone_type" {
```

Comment:
We already have such variable, it is "aws_private"

Reply:
Dropped commits from yesterday. Only updated readme.md